guides/common/modules/con_overview-of-project-integration-with-insights.adoc

```diff
@@ -0,0 +1,29 @@
+[id="overview-of-{project-context}-integration-with-insights"]
+= Overview of {Project} integration with Insights
+
+ifdef::installing-satellite-server-connected[]
+In a connected {Project} environment, you can register your {ProjectServer} with Insights or deploy the local {advisorengine}.
+endif::[]
+ifdef::installing-satellite-server-disconnected[]
+In a disconnected {Project} environment, you can install the local {advisorengine}.
+endif::[]
+
+ifdef::installing-satellite-server-connected[]
+{ProjectServer} as an Insights client::
+Connecting your {ProjectServer} to Insights as a client enables you to use the Insights dashboard to quickly identify key risks to stability, security, and performance.
++
+Note that you do not require a Red{nbsp}Hat Insights entitlement in your subscription manifest.
+endif::[]
+
+The {advisorengine}::
+The {advisorengine} analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings.
```

Comment:
This renders as "The Insights advisor in Satellite." I don't think the "the" is necessary in most uses.

```suggestion
{advisorengine} analyzes system health and configuration by applying predefined rules to a small set of local data, such as installed packages, running services, and configuration settings.
```

Perhaps {advisorengine} could be renamed to {insights_advisor_in_satellite} to discourage unnecessary articles in the future.